packages/vscode-extension/src/commands/ListObjects.ts

```diff
@@ -0,0 +1,60 @@
+import * as vscode from 'vscode';
+import SuiteCloudRunner from '../core/SuiteCloudRunner';
+import CommandsMetadataSingleton from '../service/CommandsMetadataSingleton';
+import { getRootProjectFolder, unwrapExceptionMessage } from '../util/ExtensionUtil';
+import { scloudOutput } from '../extension';
+import { MessageService } from '../service/MessageService';
+
+const objectTypes: [] = require('@oracle/netsuite-suitecloud-nodejs-cli/src/metadata/ObjectTypesMetadata');
+
+export default async function listobjects() {
+
+    const listobjectsInfo = new MessageService('listobjects');
+    const executionPath = getRootProjectFolder();
+
+    if (executionPath) {
+        const suiteCloudRunner = new SuiteCloudRunner(executionPath, CommandsMetadataSingleton.getInstance().getMetadata());
+
+        const selectedType = await vscode.window.showQuickPick(
+            objectTypes.map((objectType: any) => objectType.value.type),
```

Comment:
if you specify correctly the value of objectTypes you won't need this `: any`

Reply:
done